scripts/performance/perf_processes.py

```diff
@@ -56,10 +58,9 @@ def check_seed(seed: str):
                     type=check_seed, required=False, dest='seed')
 
 parser.add_argument('-k', '--kind',
```

Comment:
Please provide documentation with real example on how to use the script (consider multiple examples and modes)